### PR TITLE
ManagementRolesArticlesSimplified

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1578.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md [hunk 1]
```diff
@@ -183,3 +183,103 @@ Microsoft Exchange Server 2013 includes many management roles by default. The fo
...
+
+## Management role assignments
+
+For these roles to grant permissions, it must be assigned to a role assignee, which can be a role group, user, or universal security group (USG). This assignment is done using management role assignments. Role assignments link role assignees and roles together. If more than one role is assigned to a role assignee, the role assignee is granted the combination of all the permissions granted by all the assigned roles.
+
+In addition to linking role assignees to roles, role assignments can also apply custom or built-in management scopes. Management scopes control which recipient, server and database objects can be modified by role assignees. If these roles are assigned to a role assignee, but a management scope allows the role assignee only to manage certain objects based on a defined scope, the role assignee can only use the permissions granted by these roles on those specific objects. The permissions provided by these roles can't be applied to objects outside the scope defined on the role assignment. For more information about role assignments and scopes, see the following topics:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md:195]:
```suggestion
In addition to linking role assignees to roles, role assignments can also apply custom or built-in management scopes. Management scopes control which recipient, server, and database objects can be modified by role assignees. If these roles are assigned to a role assignee, but a management scope allows the role assignee only to manage certain objects based on a defined scope, the role assignee can only use the permissions granted by these roles on those specific objects. The permissions provided by these roles can't be applied to objects outside the scope defined on the role assignment. For more information about role assignments and scopes, see the following topics:
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md:195]: Please add a comma after "server" on the second line.

File: Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md [hunk 2]
```diff
@@ -183,3 +183,103 @@ Microsoft Exchange Server 2013 includes many management roles by default. The fo
...
+
+  - [Understanding management role assignments](understanding-management-role-assignments-exchange-2013-help.md)
+
+  - [Understanding management role scopes](understanding-management-role-scopes-exchange-2013-help.md)
+
+these roles are assigned to one or more role groups by default. For more information, see the "Default Management Role Assignments" section later in this topic.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md:201]:
```suggestion
These roles are assigned to one or more role groups by default. For more information, see the "Default Management Role Assignments" section later in this topic.
```

File: Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md [hunk 3]
```diff
@@ -183,3 +183,103 @@ Microsoft Exchange Server 2013 includes many management roles by default. The fo
 [View-Only Configuration role](view-only-configuration-role-exchange-2013-help.md)
 
 [View-Only Recipients role](view-only-recipients-role-exchange-2013-help.md)
+
+This management role is one of several built-in roles in the Role Based Access Control (RBAC) permissions model in Microsoft Exchange Server 2013. Management roles, which are assigned to one or more management role groups, management role assignment policies, users, or universal security groups (USG), act as a logical grouping of cmdlets or scripts that are combined to provide access to view or modify the configuration of Exchange 2013 components, such as mailbox databases, transport rules, and recipients. If a cmdlet or script and its parameters, together called a management role entry, are included on a role, that cmdlet or script and its parameters can be run by those assigned the role. For more information about management roles and management role entries, see [Understanding management roles](understanding-management-roles-exchange-2013-help.md).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md:187]:
```suggestion
This management role is one of several built-in roles in the Role Based Access Control (RBAC) permissions model in Microsoft Exchange Server 2013. Management roles, which are assigned to one or more management role groups, management role assignment policies, users, or universal security groups (USG), act as a logical grouping of cmdlets or scripts that are combined to provide access to view or modify the configuration of Exchange 2013 components, such as mailbox databases, transport rules, and recipients. If a cmdlet or script and its parameters, together called a management role entry, are included in a role, that cmdlet or script and its parameters can be run by those assigned the role. For more information about management roles and management role entries, see [Understanding management roles](understanding-management-roles-exchange-2013-help.md).
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md:187]: Please change "are included on a role" to "are included in a role".

File: Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md [hunk 4]
```diff
@@ -183,3 +183,103 @@ Microsoft Exchange Server 2013 includes many management roles by default. The fo
...
+
+If you want to view a list of role groups, users, or USGs assigned to these roles, use the following command.
+
+```powershell
+Get-ManagementRoleAssignment -Role "<role name>"
+```
+
+## Regular and delegating role assignments
+
+these roles can be assigned to role assignees using either regular or delegating role assignments. Regular role assignments grant the permissions provided by the role to the role assignee. Delegating role assignments grant the role assignee the ability to assign the role to other role assignees. For more information about regular and delegating role assignments, see [Understanding management role assignments](understanding-management-role-assignments-exchange-2013-help.md).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md:211]:
```suggestion
These roles can be assigned to role assignees using either regular or delegating role assignments. Regular role assignments grant the permissions provided by the role to the role assignee. Delegating role assignments grant the role assignee the ability to assign the role to other role assignees. For more information about regular and delegating role assignments, see [Understanding management role assignments](understanding-management-role-assignments-exchange-2013-help.md).
```

File: Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md [hunk 5]
```diff
@@ -183,3 +183,103 @@ Microsoft Exchange Server 2013 includes many management roles by default. The fo
...
+
+## Enabling or disabling role assignments
+
+By enabling or disabling a role assignment, you control whether that role assignment should be in effect. If a role assignment is disabled, the permissions granted by the associated role aren't applied to the role assignee. This is convenient if you want to temporarily remove permissions without deleting a role assignment. For more information, see [Change a role assignment](change-a-role-assignment-exchange-2013-help.md).
+
+
+## Management role customization
+
+these roles has been configured to provide a role assignee with all necessary cmdlets and parameters to manage the features and components listed in the beginning of this topic. Other roles have also been provided to enable management of other features. By adding and removing roles to and from role groups, you can create a customized permissions model without the need to customize individual management roles. For a complete list of roles, see [Built-in management roles](built-in-management-roles-exchange-2013-help.md). For more information about customizing role groups, see [Manage role groups](manage-role-groups-exchange-2013-help.md).
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md:255]:
```suggestion
These roles have been configured to provide a role assignee with all necessary cmdlets and parameters to manage the features and components listed at the beginning of this topic. Other roles have also been provided to enable management of other features. By adding and removing roles to and from role groups, you can create a customized permissions model without the need to customize individual management roles. For a complete list of roles, see [Built-in management roles](built-in-management-roles-exchange-2013-help.md). For more information about customizing role groups, see [Manage role groups](manage-role-groups-exchange-2013-help.md).
```

Comment (JohanFreelancer9) [reply Exchange/ExchangeServer2013/built-in-management-roles-exchange-2013-help.md:255]: Please correct first sentence as indicated.

**Comment by dariomws** [comment]
Thanks @JohanFreelancer9!

**Comment by scanum** [comment]
@msdmaguire PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**